tests/test_parametric_shapes/test_RotateStraightShape.py

```diff
@@ -7,6 +7,9 @@
 import pytest
 from paramak import RotateStraightShape
 
+import os
+from pathlib import Path
```

Comment:
Code inspector suggests to move imports arround.

Reply:
now sorted imports